src/AndcultureCode.CSharp.Sitefinity.Core/Interfaces/IDynamicContentRepositoryConductor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AndcultureCode.CSharp.Core.Interfaces;
+using System;
+using System.Linq;
+using System.Linq.Expressions;
+using Telerik.Sitefinity.DynamicModules.Model;
+using Telerik.Sitefinity.GenericContent.Model;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Interfaces
+{
+    public interface IDynamicContentRepositoryConductor
+    {
+        string DefaultProviderName { get; set; }
+
+        IResult<IQueryable<DynamicContent>> GetLiveContentByType(string contentType, Expression<Func<DynamicContent, bool>> filter = null);
```

Comment:
Up to you, but it might be worth adding documentation to these interfaces added in this PR.